specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/modelManagement.json

```diff
@@ -0,0 +1,4613 @@
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Model"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ModelErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/modelmanagement/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.MachineLearningServices/workspaces/{workspace}/models": {
+      "get": {
+        "tags": [
+          "Model"
+        ],
+        "summary": "Query the list of Models in a workspace.",
+        "description": "The result list can be filtered using tag and name. If no filter is passed, the query lists all the Models in the given workspace. The returned list is paginated and the count of items in each page is an optional parameter.",
+        "operationId": "MLModels_ListQuery",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceParameter"
+          },
+          {
+            "name": "name",
+            "in": "query",
+            "description": "The object name.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "tag",
+            "in": "query",
+            "description": "The object tag.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "framework",
+            "in": "query",
+            "description": "The framework.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "description",
+            "in": "query",
+            "description": "The object description.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "count",
+            "in": "query",
+            "description": "The number of items to retrieve in a page.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$skipToken",
+            "in": "query",
+            "description": "The continuation token to retrieve the next page.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "tags",
+            "in": "query",
+            "description": "A set of tags with which to filter the returned models.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "properties",
```

Comment:
What is the format of this string?

Reply:
collectionformat: "multi"

@pgmisc Refer to the link https://swagger.io/docs/specification/2-0/describing-parameters/

Reply:
Will add example with comment.